pkg/api/v1beta3/dynakube/oneagent_props.go

```diff
@@ -292,16 +292,14 @@ func (dk *DynaKube) HostGroupAsParam() string {
 }
 
 func splitArg(arg string) (key, value string) {
-	split := strings.Split(arg, "=")
```

Comment:
why not just use `split := strings.SplitN(arg, "=", 2)`

Reply:
What's wrong with `Cut`? IMHO its a bit more explicit.

Reply:
would have been less code change 😅 

but also, doing `if !found` is not different then just doing a `return argName, argValue`, because the `if !found` is true, then `argName` and `argValue` will already be empty, no?

Reply:
No, `If sep does not appear in s, cut returns s, "", false.`

Reply:
Ah, thanks for clarifying.

Are we sure that there are no arguments in the format for "--turn-on-something" ? (ie.: no "=" at all but still valid)

Reply:
But there's another issue with this code, it doesn't support arguments without values. Will fix this.

Reply:
Fixed in [7c082dd](https://github.com/Dynatrace/dynatrace-operator/pull/4502/commits/7c082ddb28b8627fdfdd9e8e539cc676c4ad0a67)